pkg/console/operator/sync_v400.go

```diff
@@ -125,21 +118,19 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	klog.V(4).Infof("sync loop 4.0.0 resources updated: %v", toUpdate)
 	klog.V(4).Infoln("-----------------------")
 
-	// the operand is in a transitional state if any of the above resources changed
-	// or if we have not settled on the desired number of replicas or deployment is not up to date.
-	if toUpdate {
-		co.ConditionResourceSyncProgressing(updatedOperatorConfig, "Changes made during sync updates, additional sync expected.")
-	} else {
+	co.HandleProgressing(updatedOperatorConfig, "ResourceSyncUpdatesInProgress", func() error {
+		if toUpdate {
+			return errors.New("Changes made during sync updates, additional sync expected.")
```

Comment:
Wont returning an error set the `Progressing` condition to `False` ?
Thought that if there needs to be a an additional sync loop, then the `Progressing` should be `True`

Reply:
Returning an `error` will set `Progressing: true` with the error message.

BUT I don't think I did the inversion on `Available!`

Leaving a note on that PR: https://github.com/openshift/console-operator/pull/272/files#r314387565